data_masks/testdata/001-expected-zero-all-primitives.pb.txt

```diff
@@ -0,0 +1,118 @@
+# api_spec.Witness proto
+
+method: {
+    meta: {
+        http: {
+            method: "POST"
+            path_template: "/v1/doggos" 
```

Comment:
I would like to see the test explicitly demonstrating redaction of a path segment.  You could have 'doggos' as a sensitive data string in the config, to re-use this sample example.